functions/Set-DbaTcpPort.ps1

```diff
@@ -116,7 +122,18 @@ function Set-DbaTcpPort {
                 if ($Pscmdlet.ShouldProcess($instance, "Setting port to $Port for IP address $IpAddress of $instance")) {
                     $netConf | Set-DbaNetworkConfiguration -Credential $Credential -EnableException:$EnableException -Confirm:$false
                 }
+
+                if (Test-Bound -ParameterName Force) {
+                    if ($PSCmdlet.ShouldProcess($instance, "Force provided, restarting Engine and Agent service for $instance on $computerFullName")) {
+                        try {
+                            $null = Stop-DbaService -ComputerName $computerFullName -InstanceName $instanceName -Type Agent, Engine
+                            $null = Start-DbaService -ComputerName $computerFullName -InstanceName $instanceName -Type Agent, Engine
```

Comment:
Why don't you use Restart-DbaService? What if the agent was not started? I think Restart takes care of this. Otherwise the agent would not be in the same state as before using this command.

Reply:
I copied it from Enable-DbaAgHadr since it is a similar config that requires a restart to take effect. Happy to use that instead if its preferred.